x/uibc/client/tests/suite.go

```diff
@@ -0,0 +1,38 @@
+//go:build experimental
+// +build experimental
+
+package tests
+
+import (
+	"github.com/cosmos/cosmos-sdk/testutil/network"
+	"github.com/stretchr/testify/suite"
```

Comment:
as noted in the workshop and in Slack, please don't use `suite`. In avg keeper I made one example how to use standard `testing` and `gotest.tools` packages.

Reply:
OK